rewrite-java/src/test/java/org/openrewrite/java/RemoveMethodInvocationsVisitorTest.java

```diff
@@ -172,7 +173,6 @@ void method() {
     }
 
     @Test
-    @ExpectedToFail
     void removeWithoutSelect() {
```

Comment:
@timtebeek After I created a fix for the new `removeLocalMethodInvocation`, I removed it because it was the same as this disabled test!